great_expectations/rule_based_profiler/parameter_builder/simple_date_format_string_parameter_builder.py

```diff
@@ -284,10 +284,9 @@ def _build_parameters(
             parameters=parameters,
         )
 
-        best_fmt_string: Optional[str] = None
-        best_ratio: float = 0.0
-
         # get best-matching datetime string that matches greater than threshold
+        best_fmt_string: str
```

Comment:
please rename to `best_format_string`

Reply:
@donaldheppner I can do it; in our expectations we have been using `fmt` as the abbreviation.  @NathanFarmer do you have any comment either way (since you developed this `ParameterBuilder` subclass)?  Thanks!